sdk/core/azure-core/src/main/java/com/azure/core/util/Configuration.java

```diff
@@ -83,6 +83,11 @@
      */
     public static final String PROPERTY_AZURE_CLIENT_CERTIFICATE_PATH = "AZURE_CLIENT_CERTIFICATE_PATH";
 
+    /**
```

Comment:
can you break this into multi-lines so it's more consistent with the other comment lengths in the same file